### PR TITLE
Shorten Intro and Entry point description

### PR DESCRIPTION
**Pull request by zastavnitskiy**
Guys, check this attempt to make the doc shorter and easier to read. I think it will improve the readability, as it removed some extra overhead without losing the important information.

If you think it's a good direction to move to, I can proceed with more edits.

### CODE REVIEW

**Comment by zastavnitskiy** [comment]
Why? It's a new sentence.

File: content/concepts/index.md [hunk 1]
```diff
@@ -7,29 +7,23 @@ contributors:
...
-
-As part of your webpack learning journey, we wrote this document aimed to give you a **high-level** overview of these concepts, while still providing links to concept specific use-cases.
+*Webpack* is a _module bundler_ for modern javascript applications. This document will give you a high-level overview of  **4 Core Concepts** you should understand before you start [configuring it.](/configuration)
 
 ## Entry
 
-webpack creates a graph of all of your application's dependencies. The starting point of this graph is known as an _entry point_. The _entry point_ tells webpack _where to start_ and follows the graph of dependencies to know _what to bundle_. You can think of your application's _entry point_ as the **contextual root** or **the first file to kick off your app**.
+Webpack creates a graph of all of your application's dependencies. The root node of dependencies graph is known as an _entry point_. The _entry point_ tells webpack _where to start_ and follows the graph of dependencies to know _what to bundle_.
```
Comment (zastavnitskiy) [inline content/concepts/index.md]: Why? It's a new sentence.

Comment (SpaceK33z) [inline content/concepts/index.md]: See https://github.com/webpack/media#name

**Comment by SpaceK33z** [comment]
@TheLarkInn, what do you think of this change?

**Comment by bebraw** [comment]
Yeah, it's probably best not to assume too much of the readers as not all are versed with graph theory. A good visualization would actually help a lot here.

**Comment by TheLarkInn** [comment]
I agree with @bebraw, if we do remove content some sort of graphic I think would be really beneficial.

**Comment by bebraw** [comment]
Can you rebase against `master`? The development happens against it now (faster deployments).

**Comment by bebraw** [comment]
Can you resolve the conflicts and do the changes requested? Thanks.

**Comment by bebraw** [comment]
I'll close this for now. Feel free to rebase and re-open if/when you have time.

**Pull request closed without merging**